.github/workflows/packages.yaml

```diff
@@ -65,3 +69,33 @@ jobs:
       with:
         name: "${{ steps.generate-artifact-properties.outputs.artifact-name }}"
         path: 'packages'
+  BuildMarinerRelease:
```

Comment:
This job should be added to the OS/arch matrix above. Does that not work?

Reply:
This doesn't work mainly because we are not building the mariner version of this on a mariner system. we need to build the Mainer version on Ubuntu-18.04 docker image, and the current matirx above did not support different builds from the container os. If we want I think i can added a build-for variable to each entry in the matirx, but these would have to be hard coded and would not be a generated combination like its current state.

Reply:
You don't need to add a new variable for every matrix entry, just for mariner with [`include`.](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations) Then map that variable to the `OS` env var, and change the scripts that fill in `$OS` based on `/etc/os-release` to only do it if `OS` isn't already set.

Reply:
changed to use the same matrix section and removed duplicate task